conda_forge_tick/audit.py

```diff
@@ -67,6 +67,7 @@
 PREFERRED_IMPORT_BY_PACKAGE_MAP = {
     "numpy": "numpy",
     "matplotlib-base": "matplotlib",
+    "qt-main": "qt",
```

Comment:
Do we want this now. Or after the migration has gone through?

Reply:
Now is fine. Nothing really can break in a big way here.

Reply:
wont this warn people to prematurely change their qt package namr

Reply:
These audits are likely not running. Also we can't run the mini migrator unless qt-main exists already.

Reply:
understood.